packages/components/psammead-figure/src/index.stories.jsx

```diff
@@ -10,10 +13,21 @@ const imageAlt =
 const imageSrc =
   'https://ichef.bbci.co.uk/news/640/cpsprodpb/439A/production/_100960371_syrians_and_asylum_v2-nc.png';
 const imageWidth = 853;
+const imageRatio = 125;
+
 storiesOf('Figure', module)
   .addDecorator(withReadme(Readme))
   .add('containing Image', () => (
     <Figure>
       <Image alt={imageAlt} src={imageSrc} width={imageWidth} />
     </Figure>
+  ))
+  .add('containing Image, ImagePlaceholder, Copyright and Caption', () => (
+    <Figure>
+      <ImagePlaceholder ratio={imageRatio}>
+        <Image alt={imageAlt} src={imageSrc} width={imageWidth} />
+        <Copyright>Copyright</Copyright>
```

Comment:
Since this is example code, it would be better to ensure this includes best practice for the offscreen text. This means using the `psammead-visually-hidden` component for Copyright and Caption, as has been updated in the Readme here: https://github.com/bbc/psammead/pull/334/files#diff-5eb481840007c7650cf4a13da9b8a8bdR39